reconcile/checkpoint.py

```diff
@@ -133,7 +133,7 @@ def report_invalid_metadata(
     """
     if dry_run:
         do_cut = partial(
-            render_template,  # type: ignore
+            render_template,
```

Comment:
Weird. Mypy was totally bitching about this partial because on the other branch of the `if` the called function has a different signature (but same return type). Glad to see they corrected it.

Reply:
I was surprised by how many of these ignores were already stale as well, which goes to show you it's good that we have a method for catching them. Thanks for the review!